$OpenBSD: patch-stm_cabal,v 1.1 2018/01/22 21:23:53 kili Exp $

Builds fine with base-4.11.

Aas usual, this is a 'metadata only' update, no new release, no new
distfile, nice for upstream, but fucking distribution package
maintainers:

http://hackage.haskell.org/package/stm-2.4.4.1/revisions/

Index: stm.cabal
--- stm.cabal.orig
+++ stm.cabal
@@ -34,7 +34,7 @@ library
         other-extensions: Safe
 
     build-depends:
-        base  >= 4.3 && < 4.10,
+        base  >= 4.3 && < 4.11,
         array >= 0.3 && < 0.6
 
     exposed-modules:
